app/views/home/index.html.haml

```diff
@@ -176,7 +176,7 @@
       .home-statistics-link.justify-content-end{style: @analytics.empty? && "visibility: hidden"}
         = link_to t("home.see_details"),'/statistics', target: "_blank"
 
-  - if @slices
+  - if slices_enabled?
```

Comment:
keept it:
`- if @slices && slices_enabled?`
to not render an empty section if no slice exists

Reply:
If we have no slices and the slices feature is enabled we still want to show that section as they can request new slices, no?

Reply:
Yes, you are correct